app/ios/iosimagepicker.cpp

```diff
@@ -0,0 +1,265 @@
+#include <QStandardPaths>
+#include <QFileDialog>
+#include <QTemporaryFile>
+#include <QUrl>
+#include <QRunnable>
+#include <QPointer>
+#include <QThreadPool>
+#include <QImageWriter>
+#include <QImageReader>
+#include "iossystemdispatcher.h"
+#include "iosimagepicker.h"
+#include <QtDebug>
+#include <QCoreApplication>
+
+#include "inpututils.h"
+
+class IOSImagePickerSaver : public QRunnable
+{
+  public:
+    QPointer<IOSImagePicker> owner;
```

Comment:
encapsulate -> move to private and rename with starting "m". (mOwner,..) create setter,getter